src/main/java/seedu/address/model/person/Appointment.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+
+
+/**
+ * Represents an Appointment attached to a Person in AddressBook
+ * Guarantees: immutable
+ */
+public class Appointment implements Comparable<Appointment> {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Appointment constraints";
```

Comment:
Seems like this was left empty. Will this be addressed in the future? If so, please leave a comment in the format `//TODO: comment`